contrib/examples/prometheus/prometheus.yml

```diff
@@ -0,0 +1,341 @@
+#
+# Prometheus will run uner the "prometheus" Service Account, create
+# access to the necessary resources
```

Comment:
What is "create access to the necessary resources"?

Reply:
I think maybe period for both as separate sentences. Still not sure why we would need the second sentence. It's rbac, it's all about access.